skaffold.yaml

```diff
@@ -31,7 +31,7 @@ deploy:
     releases:
       - name: network-orderer
         chartPath: charts/hlf-k8s
-        namespace: orderer
+        namespace: &nsord orderer
```

Comment:
In my opinion it add more complexity than usefulness but that's personal taste.

Reply:
I have no strong opinion on this, I just thought it would make it easier to change namespaces if you just have one value to change and avoid a hard to catch mistake if you change one but not the other.
But as the SA creation is namespaced you would also have to change the SA namespace in the yaml files so it makes sense to leave it plain without using references.